packages/react-native-bottom-tabs/ios/TabView/NewTabView.swift

```diff
@@ -17,6 +17,13 @@ struct NewTabView: AnyTabView {
 
           if !tabData.hidden || isFocused {
             let icon = props.icons[index]
+            var role: TabRole? {
+              if tabData.role == "search" {
+                return .search
+              }
+
+              return nil
+            }
```

Comment:
We should encapsulate this conversion into an enum same as it was done here: https://github.com/callstackincubator/react-native-bottom-tabs/blob/c1a0dd7faf88d7b6aeaa5c321cd6c4be7ba889af/packages/react-native-bottom-tabs/ios/TabViewProps.swift#L3-L24 

So its nicely separated